G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -8,15 +8,21 @@ version = v"3.5.0"
 # Collection of sources required to build GibbsSeaWater
 # note 3.05.0-4 is too old to be cross-compiled on FreeBSD
 sources = [
-    "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip" =>
-    "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c",
+    ArchiveSource(
+        "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip", "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
+export CC="cc"
+if [[ "${target}" == aarch64-apple-* ]]; then
+    export CC="clang"
+    echo $CFLAGS
+    echo $LDFLAGS
+fi
 cd $WORKSPACE/srcdir/GSW-C-*
-cc $CFLAGS -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
-cc $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm
+$CC $CFLAGS -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
+$CC -L${libdir} $LDFLAGS -fPIC -shared -o libgswteos.$dlext gsw_oceanographic_toolbox.o gsw_saar.o -lm
 mkdir -p ${libdir}
 cp libgswteos.$dlext ${libdir}
```

Comment:
Changing `cc` to `${CC}` doesn't really change anything in practice:
```suggestion
mkdir -p ${libdir}
cc -fPIC -c -O3 -Wall gsw_oceanographic_toolbox.c gsw_saar.c
cc -fPIC -shared -o "${libdir}/libgswteos.${dlext}" gsw_oceanographic_toolbox.o gsw_saar.o -lm
```